docker-compose.yaml

```diff
@@ -0,0 +1,74 @@
+version: "3.3"
```

Comment:
**FY**I, when we start a new project we prefer to use latest stable versions for docker-compose it's `3.9`